apps/web/src/components/App/Transactions/Transaction.tsx

```diff
@@ -30,7 +31,10 @@ export default function Transaction({ tx, chainId }: { tx: TransactionDetails; c
 
   return (
     <TransactionState pending={pending} success={success}>
-      <ScanLink chainId={chainId} href={getBlockExploreLink(tx.hash, 'transaction', chainId)}>
+      <ScanLink
+        useBscCoinFallback={[ChainId.BSC, ChainId.BSC_TESTNET].includes(chainId)}
```

Comment:
Can we have a variable to show which chain is supported?
Or add a function for this. No need keep copy and paste the same code

Reply:
Done